src/components/Registration/RegPeopleAndRoles.vue

```diff
@@ -105,22 +105,19 @@
     </div>
 
     <!-- Add/Edit Bus/Corp -->
-    <v-card
-      flat
-      v-if="showOrgPersonForm"
-      class="mt-8"
-      :class="{'invalid-section': getShowErrors && !isFormValid}"
-    >
-      <RegAddEditOrgPerson
-        :initialValue="currentOrgPerson"
-        :activeIndex="activeIndex"
-        :existingCompletingParty="completingParty"
-        @addEditPerson="onAddEditPerson($event)"
-        @removePerson="onRemovePerson($event)"
-        @resetEvent="resetData()"
-        @removeCompletingPartyRole="onRemoveCompletingPartyRole()"
-      />
-    </v-card>
+    <v-expand-transition>
+      <v-card flat v-if="showOrgPersonForm" class="mt-8">
```

Comment:
Transitions must wrap a code block that is conditionally rendered (v-if) or displayed (v-show). So this transition did nothing in the RegAddEditOrgPerson component itself and needs to be here instead.